docs/development/pigs/pig-xxx.rst

```diff
@@ -0,0 +1,182 @@
+.. include:: ../../references.txt
+
+.. _pig-xxx:
+
+**************************************************
+PIG X - PIG Improvement for Computing Light Curves
+**************************************************
+
+* Author: David Fidalgo, ...
+* Created: July 02, 2018
+* Accepted:
+* Status: draft
+* Discussion:
+
+Abstract
+========
+
+In this PIG we want to discuss a restructuring of the way light curves are
+computed in Gammapy.
+In general we propose to perform a spectral analysis in each of the time bins
+of the light curve to obtain the integral flux.
+For this purpose the new :class:`LightCurveEstimator` class should essentially
+be a wrapper around the *standard* spectrum analysis of Gammapy using pipeline
+classes (e.g. `gammapy.scripts.SpectrumAnalysisIACT`).
+A further input should of course be the time intervals of the light curve,
+whose computation will be done outside of the new `LightCurve` classes.
+
+It is still under discussion if the LightCurve code should prepare new
+`gammapy.data.ObservationList` s for each light curve bin or modify a
+`~gammapy.data.GTI` table, which the background/spectrum classes would
+respect.
+
+
+Detailed Description
+====================
+
+Background / What we have now
+-----------------------------
+
+The current :class:`gammapy.time.LightCurveEstimator` class takes as input a
+:class:`gammapy.spectrum.SpectrumExtraction` instance for which a list of
+:class:`gammapy.background.BackgroundEstimate` is required.
+Apart from the time intervals, the user also has to provide a
+`gammapy.spectrum.SpectralModel` that is used to compute the expected counts in
+a time bin and to scale the integral flux with respect to the excess events
+found in that time bin.
+The parameters of the spectral model are generally obtained via a spectral fit
+to the whole data sample.
+A tutorial notebook of this approach can be found `here
+<http://docs.gammapy.org/dev/notebooks/light_curve.html>`_.
+
+Drawbacks of this approach are:
+ * the user has to estimate the spectral parameters beforehand, that is before
```

Comment:
I'm not an expert in lightcurve analysis, but I always though the lightcurve point are basically flux points in time rather than energy space. So my understanding was that a global fit to the dataset always preceeds a lightcurve analysis and lateron only the model amplitude is adjusted in the time bins. In this propsal it sounds like a lightcurve analysis is always a full fit in a certain time interval. Can you specify somewhere which of the two options is the standard way?